File: exports/multi_nested_types.go
package exports

// MultiNestedTypes exported from multi_nested_types.xlsx
type MultiNestedTypes struct {
	Id        int32                        `json:"Id"`
	DictTest5 map[string]map[string]string `json:"DictTest5"`
	ListTest3 [][]string                   `json:"ListTest3"`
	ListTest4 [][]string                   `json:"ListTest4"`
	ListTest5 [][]string                   `json:"ListTest5"`
	DictTest6 map[string][]string          `json:"DictTest6"`
	DictTest7 map[string][]string          `json:"DictTest7"`
	ListTest6 []map[string]string          `json:"ListTest6"`
	ListTest7 []map[string]string          `json:"ListTest7"`
}

File: exports/MultiNestedTypes.cs
using System.Collections.Generic;

namespace Configs
{
    public class MultiNestedTypes 
    {
        public int Id { get; set; }
        public Dictionary<string, Dictionary<string, string>> DictTest5 { get; set; }
        public List<List<string>> ListTest3 { get; set; }
        public List<List<string>> ListTest4 { get; set; }
        public List<List<string>> ListTest5 { get; set; }
        public Dictionary<string, List<string>> DictTest6 { get; set; }
        public Dictionary<string, List<string>> DictTest7 { get; set; }
        public List<Dictionary<string, string>> ListTest6 { get; set; }
        public List<Dictionary<string, string>> ListTest7 { get; set; }
    }
}